docs/VnVPlan/VnVPlan.tex

```diff
@@ -85,60 +85,52 @@ \section{Symbols, Abbreviations, and Acronyms}
-VnV planning exercise.}
-
-\wss{Summarize the extras (if any) that were tackled by this project.  Extras
-can include usability testing, code walkthroughs, user documentation, formal
-proof, GenderMag personas, Design Thinking, etc.  Extras should have already
-been approved by the course instructor as included in your problem statement.
-You can use a pull request to update your extras (in TeamComposition.csv or
-Repos.csv) if your plan changes as a result of the VnV planning exercise.}
-
-\subsection{Relevant Documentation}
-
-\wss{Reference relevant documentation.  This will definitely include your SRS
-  and your other project documents (design documents, like MG, MIS, etc).  You
-  can include these even before they are written, since by the time the project
-  is done, they will be written.  You can create BibTeX entries for your
-  documents and within those entries include a hyperlink to the documents.}
-
-\citet{SRS}
-
-\wss{Don't just list the other documents.  You should explain why they are relevant and 
-how they relate to your VnV efforts.}
+  \section{2 General Information}
+
+  \subsection{2.1 Summary}
+  
+  Orbit Watch is an online platform developed by OKKM Insights to streamline and simplify satellite imagery data analysis. The software addresses the scarcity of high-quality, labeled satellite imagery datasets tailored for specific use cases across various industries such as disaster response, environmental monitoring, urban planning, and defense. Leveraging an AI-powered crowd-sourcing model, Orbit Watch enables users to label commercially available satellite images. These labeled datasets are then utilized to train custom computer vision models, enhancing accuracy and efficiency in image analysis. The platform offers a paid service for identifying objects within satellite images and distributes earnings to users who contribute to the labeling efforts.
+  
+  \subsection{2.2 Objectives}
+  
+  The primary objectives of the Verification and Validation (V\&V) plan for Orbit Watch are:
+  
+  \begin{itemize}
+      \item \textbf{Ensure High Data Accuracy}: Build confidence in the software's correctness by verifying that the system achieves high classification accuracy for objects reported in the images. This is crucial for extracting useful information and providing value to stakeholders.
+      \item \textbf{Validate System Reliability and Accessibility}: Confirm that the system is reliable with minimal downtime and is accessible remotely for both purchasers and labelers, ensuring flexibility and efficiency in usage.
+  \end{itemize}
+  
+  The following objectives are out of scope for this V\&V plan:
+  
+  \begin{itemize}
+      \item \textbf{Usability Testing (Ease of Use)}: Due to limited resources, we will not conduct extensive usability testing to verify the ease of use of the platform. We assume that the user interface adheres to standard design principles that facilitate user interaction with minimal training.
```

Comment:
Maybe remove this, I think in NFR tests and even in section 3 Usability testing is mentioned so for consistency sake I think it should either me considered an objective or removed entirely